.github/workflows/django.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches: [ "main" ]
   pull_request:
     branches: [ "main" ]
+  workflow_dispatch:
```

Comment:
Is this needed if empty?

Reply:
Yeah, this enables us to trigger the job manually within GitHub, but we don't have any additional config to pass https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch